src/backend/TrafficCourtsApi/citizen-api/Features/Lookups/LookupsService.cs

```diff
@@ -48,5 +54,10 @@ public Statute GetCountStatute(string countCode)
             if (_lookups == null) return null;
             return _lookups.statutes.FirstOrDefault(m=>m.code==Int32.Parse(countCode));
         }
+
+        public Task<LookupsAll>  GetAllLookUps()
```

Comment:
If a method is async (ie returns a Task<T>), the method names should end with Async.